src/main/java/info/wzieleziecki/Graph.java

```diff
@@ -1,22 +1,25 @@
+package info.wzieleziecki;
+
 import java.util.*;
 
-public class Graph {
+class Graph {
 
-    private List<String> wordList;
+    private List<String> wordsWithDefinedLetterCount;
 
-    public Graph(List<String> wordList) {
-        this.wordList = wordList;
+    Graph(List<String> wordsWithDefinedLetterCount) {
+        this.wordsWithDefinedLetterCount = wordsWithDefinedLetterCount;
     }
 
-    public List<String> findPath(String startWord, String endWord) {
+    List<String> findShortestPath(String startWord, String endWord) {
+
         Map<String, String> previous = new HashMap<String, String>();
         Queue<String> queue = new LinkedList<String>();
         queue.add(startWord);
 
         String word;
         while (!queue.isEmpty() && !((word = queue.remove()).equals(endWord))) {
-            List<String> nextWords = getNextWords(word);
-            for (String nextWord : nextWords) {
+            List<String> listOfPossibleNextWords = getListOfPossibleNextWordsOf(word);
+            for (String nextWord : listOfPossibleNextWords) {
```

Comment:
Ja tego typu zmienne nazywam zwykle: `possibleWords`, bo że jest to lista to widać używając tego poza tym IDE zawsze to podpowiada. Ewentualnie jeśli wybitnie chcę zaznaczyć że to lista to bym napisał `possibleNextWordsList`, co by krócej było :)

Reply:
Gotowe